llpc/util/llpcElfWriter.h

```diff
@@ -74,6 +74,22 @@ template <class Elf> class ElfWriter {
 
   static void updateMetaNote(Context *context, const ElfNote *note, ElfNote *newNote);
 
+  static size_t getRelocsCount(const SectionBuffer *pRelocSection);
```

Comment:
nit: I think that a more standard name (from the point of view of llvm) would be `numRelocs(`.

Reply:
Addressed.